docs/10-variables/02-working-with-variables/index.md

```diff
@@ -110,7 +110,7 @@ This is a shortcut that automatically sets up:
 - A change event handler on the `input` event that updates the variable when the user types
 
 :::tip
-**The `Bind to variable` dropdown is a shortcut for input fields that are not grouped**.
+**The `Bind to variable` dropdown is a shortcut for single input fields, not for groups of input fields (such as `option` inputs), or input fields that share the same `name` attribute (such as radio buttons).**
```

Comment:
This sentence gets a bit long/complicated for my tiny brain. Perhaps it would be useful to use list items to explain what it works for and what it doesn't? Also, @cullophid created this issue to improve variable binding. Hopefully we can get that fixed soon 🤞 https://github.com/nordcraftengine/nordcraft/issues/358